app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/disposal_price.scala.html

```diff
@@ -52,7 +52,7 @@
 @userKey = @{
     representativeType match {
       case Some(PersonalRepresentative) => ".personalRep"
-      case Some(PersonalRepresentativeInPeriodOfAdmin) => ".personalRep"
+      case Some(PersonalRepresentativeInPeriodOfAdmin) => ".personalRepInPeriodOfAdmin"
```

Comment:
in the case of an agent of period of admin, it'd be better if the message key being used by the cya page and this actual page used was the same - otherwise, if we ever update the message for `personalRepInPeriodOfAdmin.agent` only the cya page and not on the actual page itself